src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -14,6 +14,9 @@
                  class="c-ne__text"
                  :class="{'c-ne__input' : !readOnly }"
                  :contenteditable="!readOnly"
+                 tabindex="0"
+                 @keydown.enter.exact.prevent
+                 @keyup.enter.exact.prevent="updateEntryValue($event, entry.id)"
                  @blur="updateEntryValue($event, entry.id)"
```

Comment:
instead of updating entries on enter and blur as well, can we update entry on blur only and then on enter key press we can force the blur event

Reply:
updated